app/javascript/components/data-tables/requests-table/index.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import PropTypes from 'prop-types';
+import MiqDataTable from '../../miq-data-table';
+
+/*const TimeSinceCreation = (currentTime, creationTime) => {
```

Comment:
@Fryguy know we discussed displaying the time since the log message was created (i.e. "4 minutes ago"), but I felt like just displaying the exact time stamp was more accurate/helpful. Thoughts?

Reply:
I'm fine with exact timestamp for now...in general I usually lean towards relative time, but we can discuss as a follow up.